gcovr/templates/functions_page.html

```diff
@@ -15,12 +15,12 @@
     <tr>
       <th scope="row">Legend:</th>
       <td class="legend">
-        <span class="coverage-low">low: &gt;= 0%</span>
+        <span class="coverage-low">low: &lt; {{COVERAGE_MED}}%</span>
```

Comment:
Please don't change the direction. In the past it was this way but it's clearer to always use >=.

Reply:
Well, ok, I don't think saying ">= 0%" is very informative, but I don't really care that much. I'll revert this part.

Reply:
But it's straight forward in the legend.